Integrations/integration-rasterize.yml

```diff
@@ -5,21 +5,32 @@ name: Rasterize
 display: Rasterize
 category: Utilities
 image: data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACgAAAAoCAYAAACM/rhtAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAATRJREFUeNrsWO0NgjAQtcYBOkJHcATdRCeQDcQJcAPcQDeQDRgBN8ANaqv9UZt+Umgq3iUPEnohr+/6jguIUrrIOZaLzAMIzp7gil8QQjk4ZcvQyA+4gaHEQDBngmhCNPNqM45I1YLuuiqCSYDgL5jE1ILGDppCwTVDKe6mIAbHGmh/Rn6qgbwz05ocOyWnMuTVYn2jUfALb24jEuw88oi01voQtJ3BMrC82CPnoBwHrvplaIltMG1IzrlqNtArOV2MgqHBCT5EGZ8MN4aC4SzWC43KJMYkoQrKSlVSbi2e96Fn3WWSEIJYuBKLw6/mtx7vspb4ZFHk6FCMCALY0R8Htm/3rw/brrFDnZBqaBWM/ZJUUeokGBb2I4z//znNJPsTAQMrEMzCJI5mjUBBIDhhvAQYALha1fUnW57EAAAAAElFTkSuQmCC
-description: Rasterize URLs given to it
+description: Rasterize URLs or emails given to it
 detaileddescription: Take a URL or HTML body and create an image or PDF out of it
 configuration: []
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "true"
+  type: 8
+  required: false
 script:
   script: |-
     import os, subprocess,sys
     reload(sys)
     sys.setdefaultencoding("utf-8")
+    proxy = demisto.params()["proxy"]
+
+    if proxy:
+        http_proxy = os.environ["http_proxy"]
```

Comment:
yes it does.
checked:
2017-07-29 12:56:00.0946 info rasterize proxy settings: --proxy=127.0.0.:80 (source: /Users/shriki/Demisto/src/src/github.com/demisto/server/services/automation/pythonrunner.go:363)